tock/tock/static/js/components/notifications.js

```diff
@@ -0,0 +1,50 @@
+(function setup_notifications() {
+  if("Notification" in window) {
+    Notification.requestPermission().then(function(result) {
```

Comment:
Does this `then` block need a `catch` statement? Or I guess it will go to catch if the user doesn't grant permission, in which case, nothing has to be done?